File: examples-sdk/typescript/602-db-mongo-crud.ts
#!/usr/bin/env tsx
// 602-db-mongo-crud — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 602-db-mongo-crud.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/602-db-mongo-crud.py
#!/usr/bin/env python3
"""602-db-mongo-crud — Python SDK equivalent
Compile: vil compile --from python --input 602-db-mongo-crud.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
